### ISSUE TITLE
[Bug Report]: 8BitDo Wireless USB Adapter not working

### ISSUE THREAD

**Issue by Yersson9876** [ndeadly/MissionControl#614]
### Switch Firmware Version

16.0.2 (Latest)

### Atmosphère Version

1.5.2 (Latest)

### Mission Control Version

0.9.1 (Latest)

### Boot Method

Hekate

### Issue Description

I updated to the latest version of missioncontrol (0.9.1) and the adapter did not work, with previous versions of missioncontrol (<=0.9.0) the adapter worked normally.

**Comment by ndeadly** [comment]
Thanks for the report. It might be that the USB event filter I set up for Dualshock 3 pairing clashes with the one used by the system hid module. See if this build makes a difference, I modified the filter to be unique to only that controller.

[MissionControl-0.9.1-usb-fix-5c07b7a.zip](https://github.com/ndeadly/MissionControl/files/11302278/MissionControl-0.9.1-usb-fix-5c07b7a.zip)

**Comment by m4tze55** [comment]
Thank you for the fix.

I tried to use the 8Bitdo V2 for the first time yesterday and came across this error.
The XBOX Series X controller connected to the adapter but no input was possible.

With this fix, it now works perfectly.

Many thanks for this and have a nice sunday :-)

**Comment by Yersson9876** [comment]
Does the [MissionControl-0.9.1-usb-fix-5c07b7a.zip](https://github.com/ndeadly/MissionControl/files/11302278/MissionControl-0.9.1-usb-fix-5c07b7a.zip)
 compilation work correctly with the 8Bitdo (V1), but my generic PS3 controller does not connect. However, with the [MissionControl-0.9.1-ds3-clone-fix-bc751ce.zip](https://github.com/ndeadly/MissionControl/files/11313210/MissionControl-0.9.1-ds3-clone-fix-bc751ce.zip) compilation, both the 8Bitdo and the generic PS3 controller work. There is a small detail, after syncing the controller it works correctly, but the lights are constantly flashing, although the controller detects all inputs from both controllers.

Thank you very much, sorry for going off-topic.

**Comment by ndeadly** [comment]
I think I had already committed the usb fix by the time I made that other build for the DS3 clone, so both builds will include the same fix. 

That's interesting that your clone controller actually works without being disconnected like the other guy who started the thread. The main problem so far is that I don't have any way to tell the clones apart from the real thing. Until I can do that I won't be able to include any fixes for them in the release versions, as it will break compatibility with the real DS3 controllers. Feel free to comment on the other thread with any other details that might help me support the clones.
